src/Adapter/Profile/Employee/QueryHandler/GetEmployeeForEditingHandler.php

```diff
@@ -52,17 +68,40 @@ public function handle(GetEmployeeForEditing $query)
             throw new EmployeeNotFoundException($employeeId, sprintf('Employee with id "%s" was not found', $employeeId->getValue()));
         }
 
+        $avatarUrl = $this->getAvatarUrl($employeeId->getValue());
+
         return new EditableEmployee(
             $employeeId,
             new FirstName($employee->firstname),
             new LastName($employee->lastname),
             new Email($employee->email),
-            $employee->getImage(),
+            $avatarUrl ? $avatarUrl['path'] : $employee->getImage(),
```

Comment:
I have some trouble understanding this line 😅 

If there is an avatar URL, we use the avatar URL. Else we use the employee image.

The avatar URL and the employee image are not the same things ? 😅

Reply:
If the avatar URL doesn't exist, the image employee returns the default image (aka Preston).